vignettes/get_AUC_demo.Rmd

```diff
@@ -96,42 +127,49 @@ resp=as.numeric(str_split(signatures[5,"resp"],"\\|")[[1]])
 cutoff=signatures[5,"cutoff"]
 
 # plot all models, this is an example of negative curves 
+oldpar <- par(no.readonly = TRUE)
+on.exit(par(oldpar))   
+par(xpd = TRUE)
 output_negative <- tcplfit2_core(conc, resp, cutoff, do.plot = TRUE)
-    
 ```
 
-Take one of the model to check - exp3
+**Figure 3:** This plot depicts all fitted concentration-response curves. All curves are decreasing starting from 0 and are below x-axis. 
 
-```{r example 2 cont., fig.height = 6, fig.width = 6}
+The code chunk below calculates the AUC for exponential 3 model with the `get_AUC` function. Notice the function returns a negative AUC value, -12.92738.
```

Comment:
Suggest having the resulting value not 'manually' input result values into the text. Rather there should be a way to render these from R objects generated within the Rmd.